docker-compose.yml

```diff
@@ -1,11 +1,11 @@
 version: '3'
 services:
-  sesar:
+  sesar-rest-api:
     image: earthchemdocker/sesar-rest-api:1.0.0
     container_name: sesar-rest-api-1.0.0
     ports:
-     - "8080:8080"
+     - "80:80"
```

Comment:
Doesn't this have to be `8080:80`? I believe the app listens on 8080 but we want to expose 80 right?